mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -22,12 +24,16 @@ def remove_troublesome_chars(string: str):
     return string
 
 
-def retry_get(url, retries=3, timeout=4):
+def retry_get(url, retries=3, timeout=4, to_osmcha: bool = False):
     """Retry a query for a variable amount of tries."""
     retry = Retry(total=retries)
     with requests.Session() as session:
         session.mount("https://", HTTPAdapter(max_retries=retry))
-        return session.get(url, timeout=timeout)
+        if to_osmcha:
+            headers = {"Authorization": OSMCHA_API_KEY}
```

Comment:
this seems not to work for me locally. Maybe I'm not using the right format here. What would you expect here?